model/adjuster/sort_references.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2019 The Jaeger Authors.
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package adjuster
+
+import (
+	"sort"
+
+	"github.com/jaegertracing/jaeger/model"
+)
+
+// SortReferences returns an Adjuster that puts CHILD_OF references first.
+// This is necessary to match jaeger-ui expectations:
+// * https://github.com/jaegertracing/jaeger-ui/issues/966
+func SortReferences() Adjuster {
```

Comment:
```suggestion
func ParentReference() Adjuster {
```
I don't think we need to sort, only swap the parent into the first position.